examples/azure/build-and-deploy-multiple-artifacts.yml

```
@@ -0,0 +1,188 @@
+  value: https://hosted.mender.io
+- name: MENDER_ARTIFACT_NAME_ONE # First Mender Artifact file name
+  value: artifact_one.mender
+- name: MENDER_ARTIFACT_NAME_TWO # Second Mender Artifact file name in the `MENDER_ARTIFACTS_FOLDER` folder
+  value: artifact_two.mender
+- name: MENDER_RELEASE_NAME_ONE # The name of Mender release for Mender Artifact name `MENDER_ARTIFACT_NAME_ONE`
+  value: release-one-1.0
+- name: MENDER_RELEASE_NAME_TWO # The name of Mender release for Mender Artifact name `MENDER_ARTIFACT_NAME_TWO`
+  value: release-two-1.0
+- name: MENDER_DEVICES_GROUP_NAME_ONE # The name of Mender devices group a deployment with Mender Artifact name `MENDER_ARTIFACT_NAME_ONE` will be triggered to
+  value: group-one
+- name: MENDER_DEVICES_GROUP_NAME_TWO # The name of Mender devices group a deployment with Mender Artifact name `MENDER_ARTIFACT_NAME_TWO` will be triggered to
+  value: group-two
+
+
+resources:
+  # Use other repositories for templates
+  # https://learn.microsoft.com/en-us/azure/devops/pipelines/process/templates
+  repositories:
+    - repository: templates
+      type: github
+      name: mendersoftware/mender-ci-workflows
+      ref: refs/heads/master
+      # GitHub service connection
+      # https://learn.microsoft.com/en-us/azure/devops/pipelines/library/service-endpoints
+      endpoint: github_service_connection
+
+
+jobs:
+#
+# Builds Mender Artifacts using 'mender-artifact' tool
+#
+- job: build_mender_artifact_one
+  pool:
+    vmImage: ubuntu-latest
+  # conrainer with pre-installed mender-artifact and mender-cli tools
+  container: mendersoftware/mender-ci-tools:mender-master
+  steps:
+  # Creates Mender Artifact
+  # In this example, hello-mender.py represents a single file Python application that we have
+  # built and test in our CI and now we want to deploy into a device running Mender. We will
+  # package it as a Single File Application Update. For more information visit Mender docs:
+  # https://docs.mender.io/artifact-creation/create-an-artifact
+  - script: |
+      curl -O https://raw.githubusercontent.com/mendersoftware/mender-ci-workflows/master/examples/hello-mender.py
+      chmod 755 hello-mender.py
+      single-file-artifact-gen \
+        --device-type raspberrypi4 \
+        -o ${MENDER_ARTIFACT_NAME_ONE} \
+        -n ${MENDER_RELEASE_NAME_ONE} \
+        --software-name hello-mender \
+        --software-version 1.0 \
+        --dest-dir /usr/local/bin \
+        hello-mender.py
+    displayName: Build Mender Artifact
+  # Uploads Azure DevOps artifacts to Azure artifact storage to pass them to the further jobs
+  - task: PublishPipelineArtifact@1
+    inputs:
+      targetPath: $(MENDER_ARTIFACT_NAME_ONE)
+      artifactName: mender_artifact_one
+
+- job: build_mender_artifact_two
+  pool:
+    vmImage: ubuntu-latest
+  # conrainer with pre-installed mender-artifact and mender-cli tools
+  container: mendersoftware/mender-ci-tools:mender-master
+  steps:
+  # Creates Mender Artifact
```

I suggest using imperative wording in the headers:

```suggestion
  # Create a Mender Artifact
```